ragmac_xdem/dem_postprocessing.py

```diff
@@ -138,7 +146,10 @@ def calculate_init_stats_parallel(
 
     def _stats_wrapper(dem_path):
         """Calculate stats of a DEM in one thread."""
-        outputs = calculate_init_stats_single(dem_path, ref_dem, roi_outlines, all_outlines)
+        try:
+            outputs = calculate_init_stats_single(dem_path, ref_dem, roi_outlines, all_outlines)
+        except ValueError as e:
```

Comment:
Could you just add a comment here to specify what errors is caught here (as done in the PR's description)?

Reply:
done :)